README.md

```diff
@@ -13,11 +13,24 @@ Install with npm:
 Include it in your code:
 
 ```js
-const createGrimmService = require('@aptoma/grimm-pusher');
-
+const grimm = require('@aptoma/grimm-pusher');
+
+const options = {
+	host: 'http://grimm.example.com',
+	apiKey: 'secret',
+	// Use process decider to control when items are actually sent
+	processDecider: grimm.processDeciders.always(),
+	onSendError: console.err
```

Comment:
Maybe, I'm passing in the failed events as well. Might be good to have separate signatures depending on the error.